cache/redis_test.go

```diff
@@ -0,0 +1,116 @@
+package cache
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/grafana/dskit/flagext"
+
+	"github.com/alicebob/miniredis"
+	"github.com/go-kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+	prom_testutil "github.com/prometheus/client_golang/prometheus/testutil"
+)
+
+func TestRedisCache(t *testing.T) {
```

Comment:
Isn't a duplicate of the test we have in `redis_client_test.go`?

Reply:
It is indeed, but using the Redis cache wrapper instead of Redis client interface. 

I don't see much value in having this duplicity, so I'll remove the one in `redis_client_test.go`.